packages/ng/plg-banner/plg-banner.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="plgBanner">
+	<div class="plgBanner-icons">
+		<lu-icon class="plgBanner-icons-front" icon="transportRocket"></lu-icon>
+		<svg class="plgBanner-icons-back" aria-hidden="true" viewBox="0 0 32 32">
```

Comment:
as it will be always the same icon, what do you think about merge this icon + background on one image and add it on the CDN ? cc @jeremie-lucca

Reply:
Agree! It would increase the control we have on this icon if it changes in the future.